doc/api/permissions.md

```diff
@@ -10,6 +10,12 @@ be accessed by other modules.
   This can be used to control what modules can be accessed by third-party
   dependencies, for example.
 
+* [Process-based permissions](#process-based-permissions) control the Node.js
+  process's access to resources such as the file system or the network.
```

Comment:
I don't see policies related to `network` in the code. Is it futuristic?

Reply:
Yes, subsequent PRs.

Reply:
It might be worth actually documenting the things that are expected to be added later

Reply:
I'm afraid of documenting those and never implementing them for unexpected reasons. Perhaps I can just remove the `network` mention?

Reply:
Yup rather just remove the network mention.